blockchain/chain/src/store/chain_store.rs

```diff
@@ -35,10 +36,21 @@ const BLOCKS_PER_EPOCH: u64 = 5;
 // A cap on the size of the future_sink
 const SINK_CAP: usize = 1000;
 
+type BoxMessage = Box<dyn Message>;
+
+/// Enum for pubsub channel that defines message type variant and data contained in message type.
+#[derive(Clone, Debug)]
+pub enum HeadChange {
+    Current(Arc<Tipset>),
+    Apply(Arc<Tipset>),
+    Store,
```

Comment:
Idk my comment still remains, I remove the Store enum and everything works fine, what am I missing?

Reply:
There is no harm in including it since it copies from the original enum like structure and in terms of allocation the other oneswill probably contribute more to the the size of the allocation than this one but I can remove it

Reply:
can you link to where this variant is? I just see:
```
const (
	HCRevert  = "revert"
	HCApply   = "apply"
	HCCurrent = "current"
)
```
in Lotus

Reply:
Been starting at this for too long, I was confused sorry about that haha. Store does not need to be there